src/renderer/components/ft-list-hashtag/ft-list-hashtag.scss

```
@@ -0,0 +1,11 @@
+@use '../../scss-partials/_ft-list-item';
+
+.handle {
+  color: inherit;
+  text-decoration: none;
+}
```

Can't see the handle class used anywhere, was this meant to be title?

```suggestion
.title {
  color: inherit;
  text-decoration: none;
}
```